cloud/aws/courses/puppetize-virtual-classroom.yaml

```diff
@@ -10,12 +10,10 @@ instructorwin:
   type: "t2.medium"
   key_name: "training"
   security_group_ids:
-    - "sg-dc5409a6"
-  subnet_id: "subnet-987484fe"
+    - "sg-4d3f2b31"
```

Comment:
@binford2k The new security group allows RDP access from the ThinRDP server only at the moment. If we need to, we can open it up to public direct RDP connections to the Windows instances.

Reply:
This comment is stale now, right?

Reply:
@binford2k Yes, I opened up the rule in that SG so connections to port 3389 are accepted from anywhere.

Reply:
hmm. I must be missing something, I think. The default virtual classroom security group allows RDP from anywhere. Is something else needed?

Reply:
I didn't notice that, I guess. I can update the PR to reference sg-f3b33794 (default virtual classroom) and get rid of the other one with that one rule in it.

Reply:
@binford2k Updated the SG with a new commit